biblib/views/user_view.py

```diff
@@ -151,9 +152,18 @@ def get_libraries(cls, service_uid, absolute_uid, start=0, rows=None, sort_col="
                     owner=owner
                 )
 
-                output_libraries.append(payload)
-
-            return output_libraries
+                if (permissions and main_permission in ['owner']) or not permissions: 
```

Comment:
Where is `permissions` set? I see `permission.permissions` but not just `permissions`, but maybe I'm missing it. Either way, I have some concerns about the `or not permissions` part being lumped in with the `'owner'` permission. Depending on how `permissions` is assigned, either stick the `or not permissions` in the `elif`, or an `else`